### PR TITLE
Make devbuild upload script say the file sizes

### LINKED ISSUE #2034: Make devbuild upload script say the file sizes
to easily see in CI logs how much a job uploaded, should be specified in MiB

### PR DESCRIPTION
**Pull request by Sourciluss667**
Show filesize on file upload in scripts/uploader.rb
fix #2034

### CODE REVIEW

**Comment by Sourciluss667** [comment]
Look at CI/CircleCI: build, in ./upload_devbuilds.rb test

File: scripts/uploader.rb [hunk 1]
```diff
@@ -192,7 +192,8 @@ def object_size(hash)
 
   # Does the whole upload process
   def upload(file, url, token)
-    puts "Uploading file #{file}"
+    filesize ='%.2f' % (File.size(file).to_f / 2**20)
```
Comment (hhyyrylainen) [inline scripts/uploader.rb:195]: This should be named `file_size`. Also this text formatting type is not used anywhere else, so you should use a different text formatting type.  RubySetupSystem/SymbolUploader.rb has already print in MiB, you should copy the approach here.

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]

File: scripts/uploader.rb (hunk 1 above)
Comment (Sourciluss667) [reply scripts/uploader.rb]: I change the variable name and format output text lile in RubySetupSystem/SymbolUploader.rb

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]
The diff looks to me like this:
Kuvakaappaus - 2021-01-18 22-06-01

`'%.2f' %` is the part that needs to go

This is what is in symbol uploader:
```
RubySetupSystem/SymbolUploader.rb
201:       "with size of #{totalSize.round 2} MiB"
```
(presumably totalSize, ignore the naming as that's some old code, is converted to MiB a little above that)

**Comment by Sourciluss667** [comment]
oh okay so I just need to use round instead of '%.2f'

**Review by hhyyrylainen — APPROVED** [review]

**Pull request merged**